srcpkgs/diceware/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'diceware'
+pkgname=diceware
+version=0.2.0
+revision=1
+build_style=cargo
+wrksrc="rusty-diceware-02bf0a3e8714b936d071c7859b51c3777255c48a"
+short_desc="Commandline diceware, sans dice, written in rustlang"
+maintainer="jcgruenhage <jan.christian@gruenhage.xyz>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.com/yuvallanger/rusty-diceware"
+# This is not a tag in git yet, see https://gitlab.com/yuvallanger/rusty-diceware/issues/2
+distfiles="https://gitlab.com/yuvallanger/rusty-diceware/-/archive/02bf0a3e8714b936d071c7859b51c3777255c48a/rusty-diceware-02bf0a3e8714b936d071c7859b51c3777255c48a.tar.gz>diceware-v${version}.tar.gz"
+checksum=13a16c9d67f07e84af3a50616f7f81fea90958d80de14316458079ab103f1434
+
+post_install() {
+	vlicense LICENSE
```

Comment:
don't need to include license on GPL licensed projects

Reply:
Oh, okay. So only when xlint tells me to include it?